packages/room-server/src/user/repositories/user.repository.ts

```diff
@@ -76,6 +77,18 @@ export class UserRepository extends Repository<UserEntity> {
     return users;
   }
 
+  /**
+   * Query user info by user uuid array
+   */
+  public async selectUserBaseInfoByUuids(uuids: string[]): Promise<UserBaseInfoDto[]> {
+    return await this.find({
+      select: ['id', 'uuid', 'avatar', 'nikeName', 'color', 'isSocialNameModified'],
+      where: [{
+        uuid: In(uuids),
```

Comment:
add default condition `isDeleted: false`.

Reply:
> add default condition `isDeleted: false`.

no `isDeleted` query in the `selectUserInfoBySpaceIdAndUuids` func.
even so, should I add the default condition `isDeleted: false` ?

Reply:
Yes. The lack of condition in origin method is our mistake.